core/src/main/java/org/bitcoinj/core/AbstractManager.java

```diff
@@ -307,7 +307,9 @@ public void setFilename(String filename) {
     }
 
     public void close() {
-        shutdownAutosaveAndWait();
+        if (vFileManager != null) {
+            shutdownAutosaveAndWait();
+        }
```

Comment:
This may be called more than once in Dash Wallet and then `shutdownAutosaveAndWait` would throw an exception.